app/assets/config/application.yml

```diff
@@ -0,0 +1,3 @@
+SENDGRID_USERNAME: heyEric
```

Comment:
So it looks like you committed this file before you added it .gitignore - it happens occasionally. Use this info to fix: https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo